src/helpers/applicationUpdates.spec.ts

```diff
@@ -90,7 +90,7 @@ describe('findAndInstallServiceWorkerUpdate', () => {
     const update = jest.fn();
     const sw = {update: update as ServiceWorkerRegistration['update']} as ServiceWorkerRegistration;
 
-    let finishUpdate: () => void = () => null;
+    let finishUpdate: (value?: any) => void = () => null;
```

Comment:
since it is `any` you shouln't need to use `?`

Reply:
this means calling `finishUpdate({});` instead of `finishUpdate();`, thought it was cleaner adding `?`